controller/src/weapon.rs

```diff
@@ -68,9 +68,10 @@ define_weapons! {
         G3SG1 { id: 11, name: "G3SG1", flags: WEAPON_FLAG_TYPE_SNIPER_RIFLE },
         Galilar { id: 13, name: "Galil AR", flags: WEAPON_FLAG_TYPE_RIFLE },
         M249 { id: 14, name: "M249", flags: WEAPON_FLAG_TYPE_MACHINE_GUN },
-        M4A1 { id: 16, name: "M4A1", flags: WEAPON_FLAG_TYPE_RIFLE },
+        M4A1 { id: 16, name: "M4A4", flags: WEAPON_FLAG_TYPE_RIFLE },
```

Comment:
The enum name should also be M4A4 then :)

Reply:
In CS2 they call M4A4 also M4A1 for some reason, so I thought that you may want it to match. xD